controller/Connect.php

```diff
@@ -22,16 +22,41 @@
 namespace oat\taoOpenId\controller;
 
 
+use oat\taoOpenId\model\InvalidTokenException;
+use oat\taoOpenId\model\RelyingPartyService;
+
 class Connect extends \tao_actions_CommonModule
 {
     /**
+     * @var RelyingPartyService
+     */
+    protected $service;
+
+    public function __construct()
+    {
+
+        parent::__construct();
+        $this->service = $this->getServiceManager()->get(RelyingPartyService::SERVICE_ID);
```

Comment:
Calls to getServiceManager() in the constructor are impossible to overload, as the servicemanager wil lget injected AFTER the constructer has been called